karton/core/main.py

```diff
@@ -42,30 +154,82 @@ def process(self, task):
+    parser.add_argument("--version", action="version", version=__version__)
+    parser.add_argument("-c", "--config-file", help="Alternative configuration path")
     parser.add_argument(
-        "--delete", action="store", help="Delete persistent karton bind"
+        "-v", "--verbose", action="store_true", help="More verbose log output"
     )
-    parser.add_argument("--version", action="version", version=__version__)
-    parser.add_argument("--config-file", help="Alternative configuration path")
-    args = parser.parse_args()
 
-    config = Config(args.config_file)
+    subparsers = parser.add_subparsers(dest="command", help="sub-command help")
 
-    if args.delete:
-        karton_name = args.delete
-        print(
-            "Are you sure you want to remove binds for karton {karton_name}?\n"
-            "Type in the karton name to confirm".format(karton_name=karton_name)
-        )
-        if input().strip() == karton_name:
-            delete_bind(config, karton_name)
-        else:
-            print("abort")
+    subparsers.add_parser("list", help="List active karton binds")
 
-        return
+    delete_parser = subparsers.add_parser("delete", help="Delete a unused karton bind")
+    delete_parser.add_argument(
+        "-n", "--name", required=True, help="Karton bind identity to remove"
+    )
 
-    if args.list:
-        print_bind_list(config)
-        return
+    configure_parser = subparsers.add_parser(
+        "configure", help="Create a new configuration file"
+    )
+    configure_parser.add_argument(
+        "-f",
+        "--force",
+        action="store_true",
+        help="Overwrite the existing configuration file",
+    )
+
+    args = parser.parse_args()
+
+    if args.verbose:
+        logging.basicConfig(level=logging.DEBUG)
+    else:
+        logging.basicConfig(level=logging.INFO)
+
+    if args.command == "configure":
+        config_filename = args.config_file or "./karton.ini"
+
+        log.debug("Creating a new configuration file in %s", config_filename)
+
+        if not args.force and os.path.exists(config_filename):
+            log.error(
+                (
+                    "There's already a configuration file under %s, please delete "
+                    "it or specify a different filename using the -c argument"
+                ),
+                config_filename,
+            )
+            return
+
+        return configuration_wizard(config_filename)
+    else:
+        try:
+            config = Config(args.config_file)
+        except RuntimeError as e:
+            log.error("Error while initializing the karton config: %s", e)
+            log.error(
+                (
+                    "Please correct the configuration file or run `karton configure` "
+                    "to initialize it"
+                )
+            )
+            return
+
+        if args.command == "list":
+            return print_bind_list(config)
```

Comment:
`return`ing function that returns None looks :fish:y. Consider:
```suggestion
            print_bind_list(config)
            return
```

But actually all the returns are unnecessary. Or rather the code is a bit bipolar, either:

```
if A:
    ...
    return

if B:
    ....
    return
```

or

```
if A:
    ...
elif B:
    ....
```
why two at the same time :thinking: